examples/hello_world/lib/main.dart

```diff
@@ -4,4 +4,6 @@
 
 import 'package:flutter/widgets.dart';
 
-void main() => runApp(const Center(child: const Text('Hello, world!', textDirection: TextDirection.ltr)));
+void main() {
```

Comment:
Fat arrow syntax for `main() =>` is preferred in examples, so new users don't write any statements other than `runApp` there - which wouldn't be picked up by the hot reload.

Reply:
Woops, shouldn't have committed this